packages/pigment-css-react/tests/createExtendSxProp/runtime-createExtendSxProp.test.js

```diff
@@ -1,11 +1,10 @@
-import { describe } from 'yargs';
 import { expect } from 'chai';
 import createExtendSxProp from '../../src/createExtendSxProp';
 
 describe('createExtendSxProp', () => {
   it('return the new copy of input', () => {
     const original = { color: 'red' };
-    expect(createExtendSxProp()(original)).to.not.equal(original);
+    expect(createExtendSxProp()(original)).to.equal(original);
```

Comment:
@siriwatknp Hopefully this is correct.

Reply:
`expect(createExtendSxProp()(original)).to.not.equal(original);` is already correct (at least, it's my intention) to ensure that the result creates a new object the same as what `extendSxProp` does.

Reply:
I changed it because it was failing.

```js
function extendSxProp(props) {
  return props;
}
```

It's returning the same object. So the reference won't be different.

Reply:
Opps, then it should be the implementation that should be fixed.

```js
function extendSxProp(props) {
  return { ...props };
}
```